lib/global.g

```diff
@@ -92,6 +102,16 @@ MAKE_READ_WRITE_GLOBAL := MakeReadWriteGVar;
 
 #############################################################################
 ##
+#F  MAKE_CONSTANT_GLOBAL ( <name> ) . . .make a global variable constant
```

Comment:
add at least a space before "make". But actually there should be enough `. . . ` to right-align the text afterwards... Alternatively, just remove the whole ` . . .make a global variable constant` bit, as it is kind of redundant given the following text.

Reply:
Still would like to see that space (or, as in my other comment: Remove that line).